tests/NHSD.BuyingCatalogue.Ordering.Services.UnitTests/OrderServiceTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using AutoFixture.NUnit3;
+using FluentAssertions;
+using NHSD.BuyingCatalogue.Ordering.Api.UnitTests.AutoFixture;
+using NHSD.BuyingCatalogue.Ordering.Domain;
+using NHSD.BuyingCatalogue.Ordering.Persistence.Data;
+using NHSD.BuyingCatalogue.Ordering.Services;
```

Comment:
Is this `using` statement required?

Reply:
- Removed `using NHSD.BuyingCatalogue.Ordering.Services;`

- And below `using` statements are required

```
using NHSD.BuyingCatalogue.Ordering.Api.UnitTests.AutoFixture;
using NHSD.BuyingCatalogue.Ordering.Domain;
using NHSD.BuyingCatalogue.Ordering.Persistence.Data;
```